docs/reference/cli-reference/cli-ref-list.md

```diff
@@ -40,9 +40,13 @@ Also see [Environment variables](cli-ref-environment-variables.md)
 ## Examples
 
 ```cli
-nuget list
+nuget list //list all packages from configured feeds
```

Comment:
// is going to cause the command to fail. Check out the right way to add comments in CLI